File: app/src/main/java/com/mxt/anitrend/view/fragment/list/MediaLatestList.java
package com.mxt.anitrend.view.fragment.list;

import android.os.Bundle;
import androidx.annotation.Nullable;

import com.mxt.anitrend.model.entity.container.request.QueryContainerBuilder;
import com.mxt.anitrend.util.KeyUtil;

public class MediaLatestList extends MediaBrowseFragment {

    public static MediaLatestList newInstance(Bundle params, QueryContainerBuilder queryContainer) {
        Bundle args = new Bundle(params);
        args.putParcelable(KeyUtil.Companion.getArg_graph_params(), queryContainer);
        MediaLatestList fragment = new MediaLatestList();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setIsFilterable(false);
    }

    @Override
    public void makeRequest() {
        Bundle bundle = getViewModel().getParams();
        queryContainer.putVariable(KeyUtil.Companion.getArg_page(), getPresenter().getCurrentPage());
        bundle.putParcelable(KeyUtil.Companion.getArg_graph_params(), queryContainer);
        getViewModel().requestData(KeyUtil.Companion.getMEDIA_BROWSE_REQ(), getContext());
    }
}

File: app/src/main/java/com/mxt/anitrend/view/fragment/list/MediaLatestList.kt
package com.mxt.anitrend.view.fragment.list

import android.os.Bundle

import com.mxt.anitrend.model.entity.container.request.QueryContainerBuilder
import com.mxt.anitrend.util.KeyUtil

class MediaLatestList : MediaBrowseFragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setIsFilterable(false)
    }

    override fun makeRequest() {
        val bundle = viewModel.params
        queryContainer!!.putVariable(KeyUtil.getArg_page(), presenter.currentPage)
        bundle.putParcelable(KeyUtil.getArg_graph_params(), queryContainer)
        viewModel.requestData(KeyUtil.getMEDIA_BROWSE_REQ(), context!!)
    }

    companion object {

        override fun newInstance(params: Bundle, queryContainer: QueryContainerBuilder): MediaLatestList {
            val args = Bundle(params)
            args.putParcelable(KeyUtil.getArg_graph_params(), queryContainer)
            val fragment = MediaLatestList()
            fragment.arguments = args
            return fragment
        }
    }
}
